Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN apt-get update
 RUN apt-get install -y make python-software-properties curl
 RUN curl -sL https://deb.nodesource.com/setup_7.x | bash -
 RUN apt-get update
-RUN apt-get install -y nodejs
+RUN apt-get install -y nodejs libldap2-dev libsasl2-dev libldap2-dev libssl-dev
```

Comment:
I wonder if we should pull all of these in by default, maybe documentation only is a better approach here?

Reply:
Yes, I'm also unsure of the best way to handle these. They are required by the python ldap module.
This Dockerfile was put there to allow easy running of the tests. Without installing these in the docker image, the tests fail to run - pyldap fails to install.
I guess the travis servers have everything installed already so no issue there.
I've left this as is for now.